scripts/background.js

```diff
@@ -28,7 +28,7 @@ setInterval(() => {
                             { src: '/images/settings-icon.png', sizes: '256x256', type: 'image/png' }
                         ]
                     });
-                };
+                }
```

Comment:
May I know why you removed the semicolons?

Reply:
I have no clue why the semicolons are removed. Must of been my code editor. I'll readd them soon and do the other stuff.